GunShipAlpha/src/Enemy.java

```diff
@@ -38,4 +78,17 @@ public boolean isIntersectingLine(LineLogic line) {
     private Rectangle createBounds() {
         return new Rectangle(x, y, EnemySprite.width(), EnemySprite.height());
     }
+    public Projectile fireAtPlayer() {
+        double targetX = 400;
```

Comment:
ew - 400 is a random magic number at best, but it should also not be hard coded. get the player's location from wherever their sprite or whatever.